snooty.toml

```
@@ -30,8 +30,9 @@ version = "5.4"
 full-version = "{+version+}.0"
 mdb-server = "MongoDB Server"
 package-name-org = "mongodb-org"
-api = "https://mongodb.github.io/mongo-java-driver/{+version+}"
-core-api = "https://mongodb.github.io/mongo-java-driver/{+version+}/apidocs/mongodb-driver-core"
+api-root = "https://mongodb.github.io/mongo-java-driver/{+version+}/apidocs"
+client-api = "{+api-root+}/mongodb-driver-sync/com/mongodb/client"
+core-api = "{+api-root+}/mongodb-driver-core/com/mongodb"
```

I think condensing these into just the following is sufficient.
```suggestion
driver-api = "https://mongodb.github.io/mongo-java-driver/{+version+}/apidocs/mongodb-driver-sync/com/mongodb"
core-api = "https://mongodb.github.io/mongo-java-driver/{+version+}/apidocs/mongodb-driver-core/com/mongodb"
```

---

removed `api-root` but kept the `client` at the end of `driver-api`, since all APIs seem to belong to that namespace